pkg/proxy/origins/clickhouse/clickhouse.go

```diff
@@ -90,39 +90,34 @@ func (c *Client) Router() http.Handler {
 
 // ParseTimeRangeQuery parses the key parts of a TimeRangeQuery from the inbound HTTP Request
 func (c *Client) ParseTimeRangeQuery(r *http.Request) (*timeseries.TimeRangeQuery, error) {
-
-	trq := &timeseries.TimeRangeQuery{Extent: timeseries.Extent{}}
-	trq.TemplateURL = urls.Clone(r.URL)
-	qi := trq.TemplateURL.Query()
+	qi := r.URL.Query()
```

Comment:
ClickHouse POST queries can get a little messy -- for example, the body can be an actual query with substitutions from query parameters.  Since it's not in our current use cases (Grafana or our data access API) I'd prefer to defer that for now, although it would be nice to make it work eventually.

Reply:
However, we still want to proxy them, so I now explicitly send POSTs directly to the origin, and at least it won't be broken.